### ISSUE TITLE
[Dark Theme] Don't show list separator

### ISSUE THREAD

**Issue by hXtreme** [slapperwan/gh4a#1030]
When using dark mode on an Amoled device the grey list item separator makes parts of the screen light up.

If the list separator isn't shown/has black colour then the experience of using Dark theme on Amoled devices would improve.

**Comment by maniac103** [comment]
I don't exactly agree there. The background isn't black anyway (it's a very dark grey), and I don't see why a medium grey divider on a dark grey background would be more distracting/disturbing than a medium grey divider on a white background (light theme) ... can you expand on this assertion?

**Comment by hXtreme** [comment]
Okay, I thought that the background was black and the dividers were the reason why my screen was lit up. Let me correct the issue then to ask for black background.

**Comment by maniac103** [comment]
No need to ... please see #957 for my opinion on that. In short: won't happen, at least unless someone sends a PR.

**Comment by hXtreme** [comment]
wow! that series of issue threads became increasingly angry.

Would it be okay if I create a pr that adds a Amoled theme that is basically just inherited from dark except for background colour set to black (and maybe black dividers). That way there would be no need to mess with headers and their colours.

**Comment by maniac103** [comment]
> Would it be okay if I create a pr that adds a Amoled theme that is basically just inherited from dark except for background colour set to black (and maybe black dividers).

Yes, as long as the 'black' theme stays separate from the 'dark' one (IOW, have 4 theme options: system defined, light, dark, black). I'm not that convinced of the black (or rather, invisible) divider idea ... is there any strong predecent for this in other apps?

> That way there would be no need to mess with headers and their colours.

That's not an option for me anyway. That discussion also happened during the Holo -> Material design conversion; I'm pretty sure header colors are a moot point by now.

**Comment by yephny** [comment]
Any update on this issue?

**Comment by Lucky-069** [comment]
This is actually much needed! I can't tolerate dark theme at all.

**Comment by hXtreme** [comment]
I've switched to using GitHub official app so people should feel free to implement this since I won't be.
